requirements.txt

```diff
@@ -5,4 +5,8 @@ tensorboard
 smatch
 yacs
 torchtext>=0.9
-matplotlib
\ No newline at end of file
+matplotlib
+nltk
+flask
+flask_restful
+flask-cors
```

Comment:
Do we still need the last 3 imports? Weren't they just for the API?

Reply:
I thought you needed them, I will remove the imports.